server/data/licenceClient.ts

```diff
@@ -50,7 +50,7 @@ export class LicenceClient {
   // eslint-disable-next-line class-methods-use-this
   async getLicence(bookingId: number): Promise<CaseWithVaryVersion> {
     const query = {
-      text: `select licence, booking_id, stage, version, vary_version from licences where booking_id = $1`,
+      text: `select licence, booking_id, stage, version, vary_version, conditions_version from licences where booking_id = $1`,
```

Comment:
Has the conditions_version column already been added to the licences table?  Does it have a default value, PREVIOUS_CONDITION_VERSION perhaps, for pre-existing records?

Reply:
that's right